docs/configuration/oauth.md

```diff
@@ -0,0 +1,72 @@
+# OAuth settings
+
+Benefits can be configured to require users to authenticate with an OAuth [Open ID Connect (OIDC)](https://openid.net/connect/)
+provider, before allowing the user to begin the Eligibility Verification process.
+
+This section describes the related settings and how to configure the application to enable this feature.
+
+## Authlib
+
+!!! tldr "Authlib docs"
+
+    Read more about [configuring Authlib for Django](https://docs.authlib.org/en/latest/client/django.html)
+
+Benefits uses the open-source [Authlib](https://authlib.org/) for OAuth and OIDC client implementation. See the Authlib docs
+for more details about what features are available. Specifically, from Authlib we:
+
+1. Register an OAuth `client` using the configured [Django settings](#django-settings)
+1. Call `client.authorize_redirect()` to send the user into the OIDC server's authentication flow, with our authorization
+   callback URL
+1. Upon the user returning from the OIDC Server with an access token, call `client.authorize_access_token()` to get a validated
+   id token from the OIDC server
+
+## Environment variables
+
+!!! warning
+
+    The following environment variables are all required for OAuth configuration
+
+### `DJANGO_OAUTH_AUTHORITY`
+
+Base address of the OAuth/OIDC server to use for authorization.
+
+### `DJANGO_OAUTH_CLIENT_ID`
+
+This application's client ID, as registered with the OAuth/OIDC server.
+
+### `DJANGO_OAUTH_CLIENT_NAME`
+
+The internal label of the OAuth client within this application.
+
+See the [`OAUTH_CLIENT_NAME`](#oauthclientname) setting for more.
```

Comment:
Very small thing I noticed: the anchor link on the page is #oauth_client_name

Reply:
Thanks! It's weird, VS Code will help you fill in these anchor links to local docs, and it mostly gets them right... except for when they contain `_` underscores.

Reply:
Fixed in [29a1a30](https://github.com/cal-itp/benefits/pull/426/commits/29a1a300dfc558590d6839e3d87680de7eeae9cc)